### ISSUE TITLE
Textures not rendering in PyBullet

### ISSUE THREAD

**Issue by stephane-caron** [jrl-umi3218/jvrc_description#2]
Textures don't render properly when loading the JVRC model with:

- Pinocchio 2 + visualization in MeshCat
- PyBullet (edit, see below)

![image](https://github.com/jrl-umi3218/jvrc_description/assets/1189580/8b60cb1b-70cc-4ba6-95f1-d5f90bdf1f69)

## Expected behavior

Either textures or (if not present) colors should appear. I'm not sure if we can realize this if-then-else behavior in URDF though :thinking: 

## Reproduction steps

1. Close `robot_descriptions.py`
2. Go to the examples directory
3. Run ``python ./show_in_meshcat.py jvrc_description``
4. Check out the opened web-browser tab

## Additional context

For Pinocchio: this is likely due to Pinocchio 2 not loading textures properly. I tried in Pinocchio 3 and textures are loaded fine there:

![image](https://github.com/jrl-umi3218/jvrc_description/assets/1189580/fcfbfd7a-0c70-4fa0-960b-e0b3a3deffac)

### System

- OS: Ubuntu 22.04
- Pinocchio version: 2.7.0
- PyBullet version: 3.25

**Comment by stephane-caron** [comment]
If it's only in Pinocchio 2 we may actually skip this, as it will be fixed in future releases. I'll check if the issue happens with any other visualizer.

**Comment by stephane-caron** [comment]
OK, well I hate to be the bearer of bad news, but the same happens in PyBullet :sweat_smile: 

![image](https://github.com/jrl-umi3218/jvrc_description/assets/1189580/a34ec17b-b0e1-44db-99cb-6e30f19fb41e)

Reproduction steps: same as above, with:

```console
python ./show_in_pybullet.py jvrc_description
```

**Comment by gergondet** [comment]
I tried the pinocchio 2 + meshcat version locally and it fails indeed. If you look at the javascript console, it fails to fetch the texture files from the server. So I'm guessing this has more to do with the meshcat version or the meshcat/pinocchio integration. I stumbled upon this: https://github.com/meshcat-dev/meshcat-python/pull/112 which has some version of it integrated in https://github.com/duburcqa/jiminy/pull/533 so maybe this somehow also made its way into pinocchio 3?

I'm less familiar with bullet but the collada importer might be a little limited?

**Comment by stephane-caron** [comment]
Yes, no worries about Pinocchio 2, the texture loading fix is on its way. I don't know more about Bullet's collada importer.

Feel free to close this issue if you are OK with both outcomes (one of them being only temporary).

**Comment by stephane-caron** [comment]
This issue is now fixed in Pinocchio. I've updated the issue title accordingly.
